Cargo.toml

```diff
@@ -14,50 +14,31 @@ resolver = "2"
 
 [workspace.dependencies]
 app_dirs2 = "2.5.5"
-backtrace = "0.3.69"
 base64 = "0.22.1"
-better-panic = "0.3.0"
 bit_field = "0.10.2"
-bytes = "1.4.0"
 clap = { version = "4.4.6", features = [ "derive", "wrap_help" ] }
 criterion = { version = "0.5.1", features = [ "async_tokio" ] }
 datachannel = { version = "0.14.0", default-features = false, features = [ "tracing" ] }
-dirs = "5.0.0"
+dirs = "6.0.0"
 dunce = "1.0.3"
 futures = "0.3.28"
 if-addrs = "0.10.1"
 Inflector = "0.11.4"
-influxive-otel-atomic-obs = "=0.0.2-alpha.1"
-influxive-child-svc = "=0.0.2-alpha.1"
-influxive = "=0.0.2-alpha.1"
-lair_keystore_api = "=0.4.4"
+influxive-otel-atomic-obs = "=0.0.4-alpha.1"
+influxive = "=0.0.4-alpha.1"
 libc = "0.2.141"
 libloading = "0.8.0"
 once_cell = "1.17.1"
-openssl-sys = { version = "0.9.90", features = [ "vendored" ] }
-opentelemetry_api = { version = "=0.20.0", features = [ "metrics" ] }
 ouroboros = "0.15.6"
-parking_lot = "0.12.1"
-prometheus = "0.13.3"
 rand = "0.8.5"
-rand-utf8 = "0.0.1"
-rcgen = { version = "0.10.0", features = [ "pem" ] }
-ring = "0.16.20"
-rustls = { version = "0.20.0", features = [ "dangerous_configuration" ] }
-rustls-native-certs = "0.6.2"
-rustls-pemfile = "1.0.2"
-sbd-e2e-crypto-client = "0.0.9-alpha2"
-sbd-server = "0.0.9-alpha2"
+sbd-e2e-crypto-client = "0.0.8-alpha"
```

Comment:
Is this downgrading to 0.0.8-alpha deliberately?

Reply:
I'm gonna have to bump that soon anyway due to the [sbd PR](https://github.com/holochain/sbd/pull/42) though.

Reply:
Not deliberate. This change has been sitting on my machine and got out-of-date. Fixed -> https://github.com/holochain/tx5/pull/131/commits/99ca1dfefcf43d121bca064d1cfcfc6bd9bb18e6